From 0a7cbe735771a6d3fd3b338331c902549771f679 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Thu, 20 Apr 2023 08:40:28 +0200
Subject: [PATCH] arm: dts: bytedevkit: remove QSPI 2nd bank pinmux

Fix error message 'prop pinctrl-0 index 2 invalid phandle' when booting
from SPI NOR. Reason for the error is that the pinmux is not activated
in the ...-u-boot.dtsi.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
---
 arch/arm/dts/stm32mp157c-byteengine-m5.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi b/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
index 4eaa714d2d..ab181c0b4c 100644
--- a/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
+++ b/arch/arm/dts/stm32mp157c-byteengine-m5.dtsi
@@ -365,8 +365,8 @@
 /* NOR Flash */
 &qspi {
 	pinctrl-names = "default", "sleep";
-	pinctrl-0 = <&qspi_clk_pins_a &qspi_bk1_pins_a &qspi_bk2_pins_a>;
-	pinctrl-1 = <&qspi_clk_sleep_pins_a &qspi_bk1_sleep_pins_a &qspi_bk2_sleep_pins_a>;
+	pinctrl-0 = <&qspi_clk_pins_a &qspi_bk1_pins_a>;
+	pinctrl-1 = <&qspi_clk_sleep_pins_a &qspi_bk1_sleep_pins_a>;
 	reg = <0x58003000 0x1000>, <0x70000000 0x4000000>;
 	#address-cells = <1>;
 	#size-cells = <0>;
-- 
2.30.2

